prow-configurations/README.md

```diff
@@ -13,9 +13,19 @@ Install the following tools:
 - [gcloud](https://cloud.google.com/sdk/gcloud/)
 - OpenSSL
 
+### Provision cluster
+Follow k8s/prow documentation steps to provision cluster on GKE (https://github.com/kubernetes/test-infra/blob/master/prow/getting_started.md#create-the-cluster).
```

Comment:
``Follow [this](https://github.com/kubernetes/test-infra/blob/master/prow/getting_started.md#create-the-cluster) Kubernetes documentation steps to provision a new cluster on GKE.``